mitodl/mitxonline
# Form error messages should appear consistently and be accessible

### What are the relevant tickets?
Fix https://github.com/mitodl/hq/issues/3827

### Description (What does it do?)
Update formik
adds aria-invalid and  aria-describedby when validation errors.


### How can this be tested?
Go to profile page, try to submit the form without a required field filled out. When you click sbmit it should scroll you to the errored field, and show the error message in red.
<img width="539" alt="Screenshot 2024-04-09 at 9 12 09 AM" src="https://github.com/mitodl/mitxonline/assets/7574259/5a75efd3-151c-47a4-aeaa-7b70eb68f337">

Do the same with all forms:
- Change email,
- change password
- profile fields
- learner records page
- reset password form
- Register email
- register password
- AddlProfileFiledsForm

Then test the accessibility of the form.
Use voiceover to move through the form, try to submit the form with empty required field. The screen reader should read the error message, and focus on the field with the error message.
